app/lib/theme/getLocaleFormattedDate.ts

```diff
@@ -0,0 +1,11 @@
+import { DateTime } from "luxon";
+
+// Receives a date and returns a string with the date in a human readable format
+export const getLocaleFormattedDate = (date: string) => {
```

Comment:
we could name the input parameter "isoDate" to indicate an implicitly expected format to the users